cypress/e2e/pages/articles/testsForAMPOnly.js

```diff
@@ -1,28 +1,27 @@
 /* eslint-disable import/prefer-default-export */
-import appConfig from '../../../../src/server/utilities/serviceConfigs';
-import { getBlockData, getVideoEmbedUrl } from './helpers';
 import { ampOnly as mostReadAssertions } from '../mostReadPage/mostReadAssertions';
 
 // TODO: Remove after https://github.com/bbc/simorgh/issues/2959
 const serviceHasFigure = service =>
   ['arabic', 'news', 'pashto', 'persian', 'urdu'].includes(service);
 
+const articleHasPlayer = testAssetId =>
+  ['cgwk9w4zlg8o', 'cj7xrxz0e8zo', 'cwl08rd38l6o'].includes(testAssetId);
```

Comment:
Could we comment which environments these assets are for, and perhaps include the full URL? 

```suggestion
const articleHasPlayer = testAssetId =>
  ['cgwk9w4zlg8o',  // https:www.bbc.com/service/articles/cgwk9w4zlg8o
  'cj7xrxz0e8zo', 
  'cwl08rd38l6o'
  ].includes(testAssetId);
```

Reply:
Can this please be annotated, as it would be really useful to know which service the optimo is for?